docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -754,6 +1245,32 @@ If you want to have a customized UI, here are some hints:
 
    ![](/docu/media/10.1_CustomApplicationUI.png)
 
+## Relevant functions and events
+
+There are some functions and events which are quite relevant for the CSK approach and you should know about to understand the background of the code.
+
+Some were already explained within the other chapters. Following you will find some further explainations.
+
+Mostly all these functions and events will exist for each module per default.
```

Comment:
I do not understand the meaning of this sentence.
Do you mean all the following functions and events? but there are no events listed below.
Why do you write they will exist per default? I assume you do only mean modules provided in the CSK GitHub and not those created manually by the reader.
Suggestion: 
The modules provided in the CSK GitHub organization provide most of the mentioned functions and events per default. It is recommended to design new modules in the same way.

Reply:
I will switch the sentences, maybe it makes more sense then...